### ISSUE TITLE
Allow passing precision to faker.number.int

### ISSUE THREAD

**Issue by leon** [faker-js/faker#2186]
### Clear and concise description of the problem

In faker 7 I could specify
```
faker.datatype.number({ min: 30, max: 200, precision: 10 })
```
to get a number between 30 and 200, with an increment of 10, so: 140, 90, 180, ...

But with faker 8, the precision was removed from int.

```
faker.number.int({ min: 30, max: 200 })
```

### Suggested solution

Allow passing a precision to the int faker function

```
faker.number.int({ min: 30, max: 200, precision: 10 })
```

**Comment by ST-DDT** [comment]
Related: https://github.com/faker-js/faker/issues/1596

In that issue we already approved the step feature. I keep this issue open because it is explicitly for the step feature.

**Comment by ST-DDT** [comment]
Workaround: `faker.number.int(min/10; max/10)*10`

**Comment by ST-DDT** [comment]
Are you willing to create a PR for this?

**Comment by leon** [comment]
Sure I can have a look

**Comment by matthewmayer** [comment]
Note that the step feature in #1596 was for float, not for int. If we do it for one we should probably do it for all number methods.

**Comment by matthewmayer** [comment]
Im still a bit dubious about how useful this parameter is in reality versus just multiplying by a constant as in the workaround posted by @SD-DDT .

To get a number between 30 and 200, with an increment of 10 can be clearly described by
`10 * faker.number.int({ min: 3, max: 20 })`
or
`faker.number.int({ min: 30, max: 200, step:10 })`

But this call requires looking at the documentation to see if you will get a) {10,20,30,40,50} or b) {5,15,25,35,45}
`faker.number.int({ min: 5, max: 50, step:10 })`

I'd rather write those two options more explicitly as:
```
10 * faker.number.int({min:1, max:5})
10 * faker.number.int({min:0, max:4}) + 5
```

**Comment by leon** [comment]
After reading your comments, I did a new PR with a step param.

It feels more natural than precision, so I think we should go with that.

https://github.com/faker-js/faker/pull/2188

**Comment by matthewmayer** [comment]
I previously argued for a name something like 'multipleOf' to make it clear it doesn't work in the same way as say the HTML input element

https://github.com/faker-js/faker/pull/1855#issuecomment-1444707308

**Comment by leon** [comment]
Any progress on this?

**Comment by matthewmayer** [comment]
Could you address the comments in the open PR: https://github.com/faker-js/faker/pull/2188#discussion_r1210399970

**Comment by xDivisionByZerox** [comment]
This issue has been resolved by #2586 and was released as part of https://github.com/faker-js/faker/releases/tag/v9.0.0-alpha.0.
